AutomatedTesting/Materials/decal/airship_symbol_decal.material

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
same here, its not clear to me where this material is used

Reply:
now set as material instead of default_gray